### DISCUSSION TITLE
"Remote control"/Visual editing missing

### DISCUSSION THREAD

**Discussion by Cinderella-Man** [cc-tweaked/CC-Tweaked#1085] [Q&A] [answered]
Hello :wave: 

I'm trying to run the newest version of Minecraft(1.18.2 version + Forge 40.1 + CC Tweaked 1.100.5) running, which works fine, but I don't see any option to open a "visual editor" to program. The book I follow uses the "remote control" that allows building programs using buttons which sounds way more accessible than typing(especially for small children). Is there a way to bring visual programming back to CC Tweaked? (my assumption was that it was part of the original ComputerCraftEdu?)

Thanks in advance for any help,

Kamil

**Comment by Lupus590** [comment, marked as answer]
ComputerCraftEdu was a closed source addon for the original ComputerCraft. Its features have never been a part of opensource CC and thus are not in CC:T.

**Reply by Cinderella-Man** [reply]
Thank you for your reply. I need to admit that I wasn't aware that ComputerCraftEdu wasn't open source from the beginning. I assume that porting the "remote controller" to the CC:T won't happen/is not planned?

**Reply by Lupus590** [reply]
A couple of people have expressed interest in recreating a block-based programming language for CC(:T) but no one is working on one that I know of.
